packages/utilities/package.json

```
@@ -43,6 +43,7 @@
     "@fluentui/dom-utilities": "^2.2.14",
     "@fluentui/merge-styles": "^8.6.0",
     "@fluentui/set-version": "^8.2.14",
+    "@fluentui/react-window-provider": "^2.2.18",
```

_nit:_ Move it one line above to maintain alphabetical order

```suggestion
    "@fluentui/react-window-provider": "^2.2.18",
    "@fluentui/set-version": "^8.2.14",
```